frontend/server/src/Controllers/Contest.php

```
@@ -2904,11 +2904,13 @@ private static function validateCommonCreateOrUpdate(
             if (empty($language)) {
                 continue;
             }
+
+            /** @psalm-suppress RedundantCondition non-falsy-string for $language is always a string */
```

Is it possible to avoid suppressing the error? If necessary, instead of using the `validateInEnum` function, you can directly add the condition that is performed in that function

---

I think this piece of code verify that `$languages` variable is a subset of the `\OmegaUp\Controllers\Run::SUPPORTED_LANGUAGES()`. I used array functions to do the same thing.